src-tauri/src/settings.rs

```diff
@@ -16,6 +15,7 @@ use crate::{repo::models::Provider, types::Result};
 const DEFAULT_EMBEDDINGS_MODEL: &str = "sentence-transformers/all-MiniLM-L6-v2";
 const DEFAULT_MODEL: &str = "OpenAI/gpt-3.5-turbo";
 const SETTINGS_FILE: &str = "settings.json";
+const DEFAULT_EXEC_STEPS_LIMIT: i64 = 12;
```

Comment:
let's keep it verbose.

```suggestion
const DEFAULT_EXECUTION_STEPS_LIMIT: i64 = 12;
```